deployment/deploy.sh

```diff
@@ -14,6 +14,15 @@ then
 fi
 source $CURRENT_DIR/vars/$DEPLOY_ENV.sh
 
+# Check if push notifications will be required
+node $CURRENT_DIR/writePushNotificationFlag.js
+if [ $? != 0 ]; then
+  echo "write push notification flag script failed"
+  exit 1
+fi
+# Source ENABLE_PUSH_NOTIFICATIONS if it exists
+[ -f $CURRENT_DIR/push_notification_flag.tmp ] && source $CURRENT_DIR/push_notification_flag.tmp
```

Comment:
2 options off the top of my head. We could write a bash script that does some weird meta-programming to get a JS variable to go up a scope into the bash environment. It's possible, but it would be brittle and less clean.

Or we would have to write our devDeployConfigs in shell script/yaml/something.

I like having the json/js config for inputting stuff. But it does require writing a file that exists for a brief minute on travis.